.mergify.yml

```diff
@@ -0,0 +1,20 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - label!=not-ready
+
+pull_request_rules:
+  - name: Automatic merge on approval & CI
+    conditions:
+      - "#approved-reviews-by>=1"
+      - check-success=Ubuntu 18
+      - check-success=Ubuntu 20
+      - check-success=Windows
+      - base=main
+      - label!=not-ready
```

Comment:
Can we instead have a label "ready-for-merge", to opt-in for the automatic merge rather than opt-out from it?

Reply:
I've update the label for `ready-to-merge`